.travis.yml

```diff
@@ -1,47 +1,72 @@
-
+sudo: false
+os:
+  - linux
+  - osx
 language: cpp
-
 compiler:
  - clang
 
 env:
- - GOPATH=${TRAVIS_BUILD_DIR}/go CPPFLAGS="-I/usr/src/gmock/include" CXXFLAGS="-Wall -Werror"
+  - CXXFLAGS="-std=c++11"
```

Comment:
Actually, you can do this, it seems:

```yaml
env:
  global:
    - CXXFLAGS="-std=c++11"
    - INSTALL_DIR=$TRAVIS_BUILD_DIR/../install
    - INSTALL_DIR=$TRAVIS_BUILD_DIR/../install
```

There's a separate `matrix` subkey to do what `env` normally does otherwise?

http://docs.travis-ci.com/user/environment-variables/#Global-Variables